docs/manual/java/guide/devmode/ServiceLocator.md

```diff
@@ -52,13 +52,64 @@ The above ensures that the Service Locator knows about the `weather` service. Th
 
 Note that if the service you want to communicate with is actually a Lagom service, you may want to read the documentation for [[integrating with an external Lagom projects|MultipleBuilds]].
 
+
+# Service Gateway
```

Comment:
My intent on this PR is to get the `akka-http`/`netty` config in the docs. I noticed the friction you describe when working the change but wanted to get to the point.